system/HTTP/RedirectException.php

```diff
@@ -0,0 +1,3 @@
+<?php namespace CodeIgniter\HTTP;
+
+class RedirectException extends \Exception {}
```

Comment:
Agreed. I know it's not as "pure" as some people like it, but I like this solution, honestly. Merging.